### DISCUSSION TITLE
Feature Request: Export in "Whats app"-Format

### DISCUSSION THREAD

**Discussion by reckter** [ReagentX/imessage-exporter#215] [Ideas]
There are lots of "analyze your whatsapp chat" tools out there.
In order to use these, the exported chat, needs to follow it's Format.

I don't think this format supports replier or any fancier messages, but the basic formatting is:
```
[<date>, <time>] <name of sender>: <Text>
```

I think having this is a nice addition to this amazing tool :D

**Comment by ReagentX** [comment]
Why not just use an iMessage analyzer tool? I found a couple searching on GitHub:

https://github.com/jonluca/mimessage
https://github.com/dsouzarc/iMessageAnalyzer
